### PR TITLE
Add info about onItemClick to the onContextMenuPreparing.items parameter

### PR DESCRIPTION
**Pull request by AlexJameson**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The widget's instance.
 #include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.items): Array<Object>
-Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description.
+Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description. Each of these items also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) field.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md]: > Each of these items also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) field.

A brief description of what it allows doing would be great.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ The widget's instance.
 #include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.items): Array<Object>
-Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description.
+Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description. Each of these items also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) handler that allows you to access the clicked or tapped item's data.
```
Comment (albertov05) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md]:
```suggestion
Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description. Each item also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) event handler, which allows you to access the clicked or tapped item's data.
```

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ The widget's instance.
 #include common-ref-elementparam with { element: "widget" }
```
Comment (albertov05) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md:24]: Please change line 18 above to the following:

`The index of the column on which the context menu is invoked. For details on indexes, see the following help topic: [Column and Row Indexes](/concepts/05%20Widgets/DataGrid/15%20Columns/12%20Column%20and%20Row%20Indexes.md '/Documentation/Guide/Widgets/DataGrid/Columns/Column_and_Row_Indexes/').`

File: api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ The widget's instance.
...
 
 ##### field(e.items): Array<Object>
-Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description.
+Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description. Each of these items also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) handler that allows you to access the clicked or tapped item's data.
 
 ##### field(e.model): Object
 Model data. Available only if you use Knockout.
```
Comment (albertov05) [inline api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onContextMenuPreparing.md:30]: Please change line 36 below as follows:

`The index of the row on which the context menu is invoked. Refer to the following help topic for more information: [Column and Row Indexes](/concepts/05%20Widgets/DataGrid/15%20Columns/12%20Column%20and%20Row%20Indexes.md '/Documentation/Guide/Widgets/DataGrid/Columns/Column_and_Row_Indexes/').`

File: api-reference/10 UI Widgets/dxTreeList/1 Configuration/onContextMenuPreparing.md [hunk 5]
```diff
@@ -24,7 +24,7 @@ The widget's instance.
 #include common-ref-elementparam with { element: "widget" }
 
 ##### field(e.items): Array<Object>
-Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description.
+Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description. Each of these items also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) handler that allows you to access the clicked or tapped item's data.
```
Comment (albertov05) [inline api-reference/10 UI Widgets/dxTreeList/1 Configuration/onContextMenuPreparing.md]:
```suggestion
Items to be displayed in the context menu. Their structure is described in the [items](/api-reference/10%20UI%20Widgets/dxContextMenu/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/items/') option description. Each item also contains the [onItemClick](/Documentation/ApiReference/UI_Widgets/dxContextMenu/Configuration/#onItemClick) event handler, which allows you to access the clicked or tapped item's data.
```

Comment (albertov05) [reply api-reference/10 UI Widgets/dxTreeList/1 Configuration/onContextMenuPreparing.md]: Also, change line 36 below to the following:

`The index of the row on which the context menu is invoked. Refer to the following help topic for more information: [Column and Row Indexes](/concepts/05%20Widgets/TreeList/10%20Columns/12%20Column%20and%20Row%20Indexes.md '/Documentation/Guide/Widgets/TreeList/Columns/Column_and_Row_Indexes/').`

**Pull request merged**